src/controllers/userController.js

```diff
@@ -1,13 +1,47 @@
-/* eslint-disable require-jsdoc */
 import models from '../database/models';
+import createUserService from '../services/UserServices';
+import bcrypt from 'bcrypt';
+import jwt from 'jsonwebtoken';
 
 const { User } = models;
 
 class UserController {
-  static getAllUsers = async (req, res) => {
-    User.findAll().then((data) => {
-      res.status(200).send({ Response: data });
+  static createUser = async (req, res) => {
+    //check if a user is in the database
+    const emailExists = await User.findOne({
+      where: { email: req.body.Email },
     });
+    if (emailExists)
+      return res.status(400).json({ message: 'Email already Exists' });
+
+    //Hash the Password
+    const salt = await bcrypt.genSalt(10);
+    const hashedPassword = await bcrypt.hash(req.body.Password, salt);
+
+    try {
+      const user = await createUserService({
+        firstName: req.body.FirstName,
+        lastName: req.body.LastName,
+        email: req.body.Email,
+        userName: req.body.UserName,
+        password: hashedPassword,
+      });
```

Comment:
do you store this salt value? How will you compare the password during a login? Because the salt is needed to verify the password